### DISCUSSION TITLE
Fusion with home key project?

### DISCUSSION THREAD

**Discussion by x23piracy** [technyon/nuki_hub#409] [Ideas]
Hi,

https://github.com/rednblkx/HomeKey-ESP32

since Nuki really messed up the fw (3.9.5), wlan stability and mqtt reability i found the nuki hub project which could be a perfect combination with the home key project. Would be really cool to waste no extra esp just for that i think the power should be able to drive both project on one device which possibly makes sense or not (i am not sure) but at least it will make work to merge them (i can't, stupid me :P)

I have a Nuki 3 Pro and it's a pain with the current fw.

having the com via ble it would also save nuki power pack energy because wlan consumes a lot more 4 months compared against 6 months (wlan <> ble)

Cust my 50 Cent...

Best Regards X23

**Comment by iranl** [comment]
Asked and answered: https://github.com/technyon/nuki_hub/issues/337
